lib/compiler/src/beam_ssa_type.erl

```diff
@@ -1671,6 +1671,18 @@ type(call, [#b_literal{val=Fun} | Args], _Anno, _Ts, _Ds) ->
             %% arguments is wrong.
             none
     end;
+type(extract, [V, #b_literal{val=Idx}], _Anno, _Ts, Ds) ->
+    case map_get(V, Ds) of
+        #b_set{op=landingpad} when Idx =:= 0 ->
+            %% Class
+            #t_atom{elements=[error,exit,throw]};
+        #b_set{op=landingpad} when Idx =:= 1 ->
+            %% Reason
+            any;
+        #b_set{op=landingpad} when Idx =:= 2 ->
+            %% Stack trace
+            #t_list{}
```

Comment:
The raw stacktrace is an opaque data type that the compiler should not know anything about. The type should be `any`. (There is no difference in `diffable` if I change the type to `any`.)

Reply:
Good point, I'll fix it here and in the validator.